CMakeLists.txt

```diff
@@ -107,11 +194,23 @@ macro(nbl_adjust_flags)
 			"$<$<CONFIG:DEBUG>:-Wall>"
 			"$<$<CONFIG:DEBUG>:-fno-omit-frame-pointer>"
 			"$<$<CONFIG:DEBUG>:-fstack-protector-strong>"
-			#"$<$<CONFIG:RELEASE>:-O3>" # added by default and apparently cmake is not clever enough to filter duplicates for this one
-			"$<$<CONFIG:RELEASE>:-fexpensive-optimizations>"
 		)
+		if(NOT CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+			if (NOT NBL_BUILD_CEGUI)
+				add_compile_options(-fno-exceptions)
+			endif()
```

Comment:
clang doesn't have a no exceptions flag?

Reply:
what why?

Reply:
ah, `if(NOT CMAKE_CXX_COMPILER_ID STREQUAL "Clang")`
i have no idea tbh, it was always here